src/style/panel.less

```diff
@@ -329,6 +329,28 @@
     .generate-columns(@n, (@i + 1));
 }
 
+/* Japanese Specific Label Sizings */
+.ja .label__medium.column-2.blur-label {
```

Comment:
Also, these class names should have a prefix. Something like `.language__ja` and `.language__de`.